packages/framework/package.json

```diff
@@ -104,7 +104,7 @@
     "zod-to-json-schema": "^3.23.0"
   },
   "dependencies": {
-    "@novu/shared": "0.24.2",
+    "@novu/shared": "workspace:*",
```

Comment:
Should be checked-in as `workspace:*`, but modified prior to publish.

Reply:
How is itmodified?

Reply:
Manually for now, but `pnpm version` has a capability to replace it, publish, then revert it automatically